docs/operator-manual/airgap/01-install.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_label: "Installation"
+title: ""
+---
+
+# Airgap installation
+
+This guide will show you how to install Kubewarden in air-gapped environments. In an air-gapped installation of Kubewarden, 
+you will need a private OCI registry that is located somewhere accessible by your kubernetes cluster. Kubewarden Policies 
+are WebAssembly modules, therefore they can be stored inside of an OCI compliant registry as OCI artifacts.
+You need to add Kubewarden's images and policies to this OCI registry. Let's see how to do that.
+
+## Save container images in your workstation
+
+1. Download `kubewarden-images.txt` from the Kubewarden [release page](https://github.com/kubewarden/helm-charts/releases/). Alternatively, you can find `imagelist.txt` and `policylist.txt` files shipped inside the helm charts, that contain the used container images and policy wasm modules respectively.
+:::note
+Optionally, you can verify the signatures of the [helm charts](https://docs.kubewarden.io/security/verifying-kubewarden#helm-charts) and [container images](https://docs.kubewarden.io/security/verifying-kubewarden#container-images)
+:::
+2. Add `cert-manager` if it is not available in your private registry. 
+```
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm pull jetstack/cert-manager --version v1.7.1
+helm template ./cert-manager-v1.7.1.tgz | awk '$1 ~ /image:/ {print $2}' | sed s/\"//g >> ./kubewarden-images.txt
+```
+3. Download [kubewarden-save-images.sh](https://github.com/kubewarden/utils/blob/main/scripts/kubewarden-save-images.sh) 
+and [kubewarden-load-images.sh](https://github.com/kubewarden/utils/blob/main/scripts/kubewarden-load-images.sh) from the Kubewarden utils repository.
+4. Save Kubewarden container images into a tar.gz file:
+```
+./kubewarden-save-images.sh --image-list ./kubewarden-images.txt --images kubewarden-images.tar.gz
+```
+Docker begins pulling the images used for an airgap install. Be patient. This process takes a few minutes. 
+When the process completes, your current directory will output a tarball named `kubewarden-images.tar.gz`. Check that the output is in the directory.
+
+## Save policies in your workstation
+
+[kwctl](https://github.com/kubewarden/kwctl/releases) version v1.3.1 or above is required for this step.
+
+1. Add all the policies you want to use in a `policies.txt` file. 
+Example: (TODO add link to default policies.txt once it is available) 
+```
+registry://ghcr.io/kubewarden/policies/pod-privileged:v0.2.2
+registry://ghcr.io/kubewarden/policies/safe-labels:v0.1.5
+```
+2. Download `kubewarden-save-policies.sh` and `kubewarden-load-policies.sh` from the [kwctl repository](https://github.com/kubewarden/kwctl/tree/main/scripts)
```

Comment:
Aren't we going to add this file to the GH release artifacts? I would do that (add the file to the release) and point to the release page instead of the `main` branch

Reply:
done in https://github.com/kubewarden/docs/pull/150/commits/a0514f49871192376d6774e8a11e5e287acb784a 
this [PR](https://github.com/kubewarden/kwctl/pull/364) adds the scripts to the kwctl release